examples/multiple-google-maps/app.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {FunctionComponent, useState, useCallback} from 'react';
+import {GoogleMapsProvider} from '@ubilabs/google-maps-react-hooks';
+
+import MapCanvas from './components/map-canvas/map-canvas';
+
+import {GOOGLE_MAPS_API_KEY} from '../constants';
+
+import './main.module.css';
+
+const basicMapOptions = {
```

Comment:
Would it make sense to keep the `mapOptions` in an extra constants (or config) file, to have less here?